pings.yaml

```diff
@@ -61,3 +61,19 @@ demographics:
     - https://github.com/mozilla-rally/rally-core-addon/pull/139#issuecomment-736024232
   notification_emails:
     - than@mozilla.com
+
+uninstall-deletion:
```

Comment:
Will this generate a `metrics.uuid.rally.id` field? The new decoder requires enveloped messages to contain this field (https://github.com/mozilla/gcp-ingestion/blob/caf1fcb5aec06f40ab8ee1c67a33df0abe29247c/ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/rally/DecryptRallyPayloads.java#L208-L213), and I want to make sure it doesn't get thrown into additional_properties.

Reply:
good catch, I forgot to mention this in the `metrics.yaml` file.